content/en/docs/setup/install/istioctl/index.md

```diff
@@ -526,6 +526,139 @@ Some parameters will temporarily exist in both the Helm and `IstioOperator` APIs
 namespaces and enablement settings. The Istio community recommends using the `IstioOperator` API as it is more
 consistent, is validated, and follows the [community graduation process](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE-CHECKLIST.md#feature-lifecycle-checklist).
 
+## Advanced install customization
+
+### Customizing external charts and profiles
+
+`istioctl` `install`, `manifest generate` and `profile` commands can use any of the following sources for charts and
+profiles:
+
+- compiled in charts. This is the default if no `--charts` option is set. The compiled in charts are the same as those
+in the `manifests/` directory in Istio release `.tgz`.
+- charts in the local file system e.g. `istioctl install --charts istio-1.6.0/manifests`
+- charts in GitHub e.g. `istioctl install --charts https://github.com/istio/istio/releases/download/1.6.0/istio-1.6.0-linux-arm64.tar.gz`
+
+Local file system charts and profiles can be customized by editing the files in the `manifests/`. For extensive changes,
+it's recommended to make a copy of this directory and make changes to the copy. `istioctl` requires the same file layout
+as in `manifests` to be preserved.
```

Comment:
```suggestion
of the `manifests` file must be preserved.
```

Reply:
it should be: of the `manifests` directory must be preserved.

Reply:
Ah, sorry. See updated suggestion.